app/components/AppsNav.tsx

```diff
@@ -86,7 +90,7 @@ export default class AppsNav extends Vue {
     }
     if (el.scrollHeight - el.scrollTop === el.clientHeight) {
       this.downArrowVisible = false;
-    } else {
+    } else if (el.scrollHeight > el.clientHeight) {
```

Comment:
I think this means the arrow won't appear if you resize the window smaller.

Reply:
this is true, but if you start scrolling it'll appear immediately. a resize handler is a little tricky here since it's like the 3rd or 4th child component down from the main window

Reply:
Seems acceptable then.